src/Resources/config/routing.yml

```diff
@@ -15,3 +15,17 @@ contao_backend_redirect:
 contao_frontend:
     resource: .
     type: contao_frontend
+
+contao_root:
```

Comment:
Isn't `contao_frontend` actually the catch all route? At least there is a comment saying "The fallback route must be the last one!" and now the route is no longer the last one :)